File: automationCucumberishCucumberTests/automationCucumberishCucumberTests.m
#import <Foundation/Foundation.h>
#import "automationCucumberishCucumberTests-Swift.h"
#import <XCTest/XCTest.h>


__attribute__((constructor))
void CucumberishInit()
{
    [automationCucumberishCucumberTests CucumberishSwiftInit];

}

File: automationCucumberishCucumberTests/automationCucumberishCucumberTests.swift
import Foundation
import Cucumberish

class automationCucumberishCucumberTests: NSObject {
    
    @objc class func CucumberishSwiftInit()
    {
        var application : XCUIApplication!
        beforeStart { () -> Void in
            application = XCUIApplication()
            // StepDefinitions:
            Background().backgroundSteps()
            ExampleSteps().example()
            // Launch App
            application.launch()
        }
               
        let bundle = Bundle(for: automationCucumberishCucumberTests.self)

        Cucumberish.executeFeatures(inDirectory: "Features", from: bundle, includeTags:nil, excludeTags: nil)
        
        before { (scenario: CCIScenarioDefinition?) in
            WiremockClient().deleteRequests()
            WiremockClient().deleteStubs()
        }
    }
}
